docs/resources/associated_site.md

```diff
@@ -0,0 +1,170 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Multi-Site"
+layout: "aci"
+page_title: "ACI: aci_associated_site"
+sidebar_current: "docs-aci-resource-aci_associated_site"
+description: |-
+  Manages ACI Associated Site
+---
+
+# aci_associated_site #
+
+Manages ACI Associated Site
+
+
```

Comment:
should we add a note/warning with some additional information regarding the class since this is not a documented class and used only for specific use-case, which can be dangerous when mis-used?